conf/server/plugin/ds_sqlite.conf

```diff
@@ -3,4 +3,6 @@ pluginCmd = "plugin/server/datastore-sqlite/datastore-sqlite"
 pluginChecksum = ""
 enabled = true
 pluginType = "DataStore"
-pluginData {}
+pluginData {
+	file_name = "./conf/server/spire.db"
```

Comment:
Agreed... I struggled a little trying to come up with the correct location for this.

I think we need a general directory for runtime file storage... use cases like attestation nonce come to mind. This seems like it may fit the bill as well... perhaps I'll create a top-level `data` directory, and stick it in there with a name like `datastore_sqlite.db` or `datastore.sqlite3`? Would that suffice?

Reply:
Yes, that looks good!